dataland-frontend/src/components/resources/companiesOnlySearch/CompaniesOnlySearchBar.vue

```diff
@@ -83,20 +80,22 @@ export default defineComponent({
      * @param event object containing the stored company
      * @param event.value the stored company object
      */
-    pushToChooseFrameworkForDataUploadPageForItem(event: { value: StoredCompany }) {
-      void this.$router.push(`/companies/${event.value.companyId}/frameworks/upload`);
+    async pushToChooseFrameworkForDataUploadPageForItem(event: AutoCompleteItemSelectEvent) {
+      await this.$router.push(
+        `/companies/${assertDefined(assertDefined(event.value as CompanyIdAndName).companyId)}/frameworks/upload`
```

Comment:
Beide `assertDefined` sind nicht notwendig. Bei mir in der IDE ist typescript auch ohne happy.

Reply:
Der Linter aber nicht. Ich verstehe es auch nicht.

Reply:
Denke das liegt an der Optionalität der companyId hier:

![image](https://github.com/d-fine/Dataland/assets/99255296/c5836d10-59c9-44d0-a267-2ef35071bcb9)


Wir können doch beide Felder guten Gewissens mandatory machen meines Erachtens, oder?
Dann sollte der linter hier auch nicht mehr meckern.

Reply:
Done.

Reply:
👍